packages/mask/src/social-network-adaptor/twitter.com/injection/PostActions/index.tsx

```diff
@@ -1,9 +1,18 @@
-import type { PostInfo } from '@masknet/plugin-infra'
+import { NetworkPluginID, PostInfo } from '@masknet/plugin-infra'
 import { Flags } from '../../../../../shared'
+import { pluginIDSettings } from '../../../../settings/settings'
 import { createPostActionsInjector } from '../../../../social-network/defaults/inject/PostActions'
 
-export function injectPostActionsAtTwitter(signal: AbortSignal, current: PostInfo) {
+export function injectPostActionsAtTwitter(signal: AbortSignal, postInfo: PostInfo) {
     if (!Flags.post_actions_enabled) return
+    if (pluginIDSettings.value !== NetworkPluginID.PLUGIN_EVM && postInfo.actionsElement) {
```

Comment:
It's okay to inject the actions bar under any network. If no plugin wants to render in it, then it renders nothing.

Reply:
> It's okay to inject the actions bar under any network

A blank area will be leaved there.